app/src/main/java/com/gholem/moneylab/domain/model/AddNextTransaction.kt

```diff
@@ -0,0 +1,18 @@
+package com.gholem.moneylab.domain.model
+
+open class AddNextTransaction {
```

Comment:
This should be `sealed class` instead of `open class`. Also the name is still not correct I think - this represent an item in the adapter so the name could be `AddTransactionItem`